of-builder/main.go

```diff
@@ -168,9 +169,12 @@ func build(w http.ResponseWriter, r *http.Request) ([]byte, error) {
 				if v.Completed != nil {
 					msg = fmt.Sprintf("v: %s %s %.2fs", v.Started, v.Name, v.Completed.Sub(*v.Started).Seconds())
 				} else {
-					startedVal := "(no timestamp available)"
+					startedVal := ""
 					if v.Started != nil {
 						startedVal = v.Started.String()
+					} else {
+						tnow := time.Now()
+						startedVal = tnow.Format("0000-00-00 23:59:59.999999999 +0000 UTC")
```

Comment:
Not sure we need the format piece. This is just using `.String()` on a `time.Time` struct. Can you update please?

Reply:
`tnow.Format` formats the time in less precision.

Reply:
@alexellis sorry for picking this one late. Do you mean to apply `time.RFC3339` to format time in all logs `v.Started` and the `tnow` ?